multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangareader/WPMangaReaderGenerator.kt

```diff
@@ -29,7 +29,7 @@ class WPMangaReaderGenerator : ThemeSourceGenerator {
         SingleLang("Skull Scans", "https://www.skullscans.com", "en"),
         SingleLang("Luminous Scans", "https://www.luminousscans.com", "en"),
         SingleLang("Azure Scans", "https://azuremanga.com", "en"),
-        SingleLang("GS Nation", "https://gs-nation.fr", "fr", overrideVersionCode = 1),
+        SingleLang("ReaperScans.fr", "https://reaperscans.fr/", "fr", className  = "ReaperScansFR", pkgName = "gsnation", overrideVersionCode = 2),
```

Comment:
```suggestion
        SingleLang("ReaperScans.fr", "https://reaperscans.fr", "fr", className  = "ReaperScansFR", pkgName = "gsnation", overrideVersionCode = 2),
```
Having that last slash can break the source